src/elements/content-sidebar/activity-feed/common/activity-message/CollapsableMessage.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import ToggleMoreLessButton from './ToggleMoreLessButton';
+import './ActivityMessage.scss';
+
+type Props = {
+    children: React.Node,
+};
```

Comment:
Do we need to define children in the props? I think React components come with children built into the props

Reply:
We need. https://flow.org/en/docs/react/children/